docs/docs/operator/bill-of-behavior/quickstart.md

```
@@ -0,0 +1,273 @@
+#  egress-kube-dns   egress-chain-postgres                  # DNS + the database, nothing else
+```
+ -->
+
+Read the raw alerts — **one JSON object per rule violation**:
+
+```bash
+kubectl -n kubescape logs -l app=node-agent --tail=200 | grep '"RuleID":"R' \
+  | jq -c '{RuleID, alertName: .BaseRuntimeMetadata.alertName, args: .BaseRuntimeMetadata.arguments}'
+```
+
+#### R0001 — unexpected process log4shell
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)